teps/0096-pipelines-v1-api.md

```diff
@@ -174,11 +182,107 @@ V1 work should prioritize stabilizing existing beta features that support the [u
 If any gaps in current features are identified, they should be V1 blockers only if mission critical to common use cases.
 
 ## API Definition
+The current Tekton Pipelines API [definition](https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md#what-is-the-api) includes:
+
+- CRDs, including their spec and status fields
+- step ordering and naming of step containers within the Task status
+- labels propagated from PipelineRuns to TaskRuns and TaskRuns to Pods
+- the /workspace and /tekton/results [directories](https://github.com/tektoncd/pipeline/blob/main/docs/tasks.md#reserved-directories) inside containers started by Tekton
+- the interfaces of the images built by Tekton pipelines
+
+This policy should be updated to include Tekton metrics as part of the API. No other changes are proposed.
 
 ## Features Included
+### Documentation
+
+- Update the compatibility policy to reflect this TEP.
+- Provide a migration guide from v1beta1 API to v1 API.
+- Track documentation updates outside of this TEP (e.g. in a Github project or milestones).
+
+### Production Readiness
+
+- Implement [TEP-0036: Start measuring performance](https://github.com/tektoncd/community/blob/main/teps/0036-start-measuring-tekton-pipelines-performance.md).
+- Implement [TEP-0073: Simplify metrics](https://github.com/tektoncd/community/blob/main/teps/0073-simplify-metrics.md).
+  - [TEP-0006: Tekton metrics](./0006-tekton-metrics.md) is out of scope because it discusses adding more granular metrics. More granular metrics may be added later.
+- Ensure metrics are accurate and mark all metrics as "alpha" or "stable". 
+
+### Stability
+
+- Track bug fixes and small backwards incompatible changes outside of this TEP (e.g. in a Github project or milestones).
+- Implement conversion webhook for v1beta1 objects.
+
+#### CRD Stability Levels
+
+- The Task, TaskRun, Pipeline, and PipelineRun CRDs should be stabilized.
```

Comment:
What does stabilized mean here?

Reply:
updated to "moved to "stable" stability level"